php/class-customize-snapshot-manager.php

```diff
@@ -1480,6 +1481,30 @@ public function render_templates() {
 				<p>{{ data.message }}</p>
 			</div>
 		</script>
+
+		<script type="text/html" id="tmpl-snapshot-conflict-button">
+			<?php $title_text = esc_html__( 'Potential Snapshot conflicts (click to expand)', 'customize-snapshots' ); ?>
+			<# id= data.setting_id.replace( /]/g, '' ).split( '[' ).filter( Boolean ).join( '-' ); #>
+			<a href="#TB_inline?width=600&height=550&inlineId=snapshot-conflicts-{{id}}" class="dashicons dashicons-warning thickbox snapshot-conflicts-button" title="<?php echo $title_text; ?>"></a>
+		</script>
+
+		<script type="text/html" id="tmpl-snapshot-conflict">
+			<# id= data.setting_id.replace( /]/g, '' ).split( '[' ).filter( Boolean ).join( '-' ); #>
+			<div id="snapshot-conflicts-{{id}}" class="snapshot-conflict-thickbox-content">
+				    <# _.each( data.conflicts, function( setting ) { #>
+						<details>
+							<summary>
+								<code>{{setting.name}}</code>
+								<a href="{{setting.editLink}}">
+									(<?php _e( 'edit', 'customize-snapshots' ); ?>)
```

Comment:
Instead of “edit” how about “inspect”? Also, the parents should be _inside_ the translated string, and best to use `esc_html_e()` here.